tests/par-test.cc

```diff
@@ -37,6 +43,9 @@ FitFunc fsphere = [](const double *x, const int N)
 
 int main(int argc, char *argv[])
 {
+#ifdef HAVE_TBB
+  static tbb::task_scheduler_init init;
+#endif
```

Comment:
Shouldnt that go somewhere in cmaes lib rather than in the test ?

Reply:
> Shouldnt that go somewhere in cmaes lib rather than in the test ? Here third party libs would still need to link to tbb.

Not really. Quoting Intel's [documentation](https://software.intel.com/en-us/node/506296):

``Using task_scheduler_init is optional in Intel® Threading Building Blocks (Intel® TBB) 2.2. By default, Intel TBB 2.2 automatically creates a task scheduler the first time that a thread uses task scheduling services and destroys it when the last such thread exits.``

I just put it for completeness.

Reply:
Does it really work without ?

Reply:
> Does it really work without ?

Yes.. You can try it..

Reply:
Yes.. I tried..

Reply:
Ok